app/src/main/java/chat/rocket/android/dagger/module/AppModule.kt

```diff
@@ -188,12 +188,39 @@ class AppModule {
     @Provides
     @Singleton
     fun provideMoshi(): Moshi {
-        return  Moshi.Builder().add(AppJsonAdapterFactory.INSTANCE).build()
+        return Moshi.Builder().add(AppJsonAdapterFactory.INSTANCE).build()
     }
 
     @Provides
     @Singleton
     fun provideMultiServerTokenRepository(repository: LocalRepository, moshi: Moshi): MultiServerTokenRepository {
         return SharedPreferencesMultiServerTokenRepository(repository, moshi)
     }
+
+    @Provides
+    @Singleton
+    fun provideMessageRepository(): MessagesRepository {
+        return MemoryMessagesRepository()
+    }
+
+    @Provides
+    @Singleton
+    fun provideConfiguration(context: Application, client: OkHttpClient): SpannableConfiguration {
+        return SpannableConfiguration.builder(context)
+                .asyncDrawableLoader(AsyncDrawableLoader.builder()
+                        .client(client)
+                        .executorService(Executors.newCachedThreadPool())
+                        .resources(context.resources)
+                        .build())
+                .theme(SpannableTheme.builder()
+                        .linkColor(Color.BLUE)
```

Comment:
Is it possible to use our `colorAccent` here?
https://github.com/RocketChat/Rocket.Chat.Android/blob/d31b9ea1262d750a826e526bb946b7cd8810c797/app/src/main/res/layout/fragment_authentication_log_in.xml#L56